spec/features/saml/saml_spec.rb

```diff
@@ -76,6 +89,10 @@
         expect(xmldoc.response_assertion_nodeset.length).to eq(1)
       end
 
+      it 'explicitly disables encryption' do
+        expect(xmldoc.original_encrypted?).to eq false
+      end
```

Comment:
Should this perhaps go under a separate context with a title like `service provider explicitly disables encryption` to make it clearer why it is expected for the encryption to be false? Otherwise, under this context, it makes it sound like encryption should always be disabled when a user gets a well-formed signed assertion.